src/components/PaymentInfoLine.vue

```diff
@@ -1,17 +1,95 @@
 <template>
     <div class="info-line" :class="{ 'inverse-theme': theme === constructor.Themes.INVERSE }">
-        <div class="amounts">
+        <div class="amounts"
+            @mouseenter="$refs['price-tooltip'] && $refs['price-tooltip'].show()"
+            @mouseleave="$refs['price-tooltip'] && $refs['price-tooltip'].hide()"
+            @click="$refs['price-tooltip'] && Date.now() - lastTooltipToggle > 200 && $refs['price-tooltip'].toggle()"
+        >
             <Amount
                 :currency="cryptoAmount.currency"
                 :amount="cryptoAmount.amount"
                 :currencyDecimals="cryptoAmount.decimals"
                 :minDecimals="0"
                 :maxDecimals="Math.min(4, cryptoAmount.decimals)"
             />
-            <FiatAmount v-if="fiatAmount" class="fiat-amount"
-                :currency="fiatAmount.currency"
-                :amount="fiatAmount.amount"
-            />
+            <Tooltip ref="price-tooltip"
+                v-if="fiatAmount"
+                :container="$parent"
+                preferredPosition="bottom left"
+                :margin="{ left: 8 }"
+                :styles="{
+                    minWidth: '34.75rem',
+                    padding: '2rem',
+                    lineHeight: 1.3,
+                }"
+                :theme="theme"
+                @show="onPriceTooltipToggle(true)"
+                @hide="onPriceTooltipToggle(false)"
+                @click.native.stop
+                class="price-tooltip"
+            >
+                <template v-slot:trigger>
+                    <FiatAmount :currency="fiatAmount.currency" :amount="fiatAmount.amount" />
+                </template>
+                <template v-slot:default>
+                    <div class="price-breakdown">
+                        <label>Order amount</label>
+                        <FiatAmount :currency="fiatAmount.currency" :amount="fiatAmount.amount" />
+                        <template v-if="vendorMarkup || vendorMarkup === 0">
```

Comment:
Was it agreed with the designers that the vendor markup is even displayed when 0%?

Reply:
Several implementation decisions were taken without involvement of the designers.
They have unfortunately been very silent when it came to details of the tooltip.
I asked them to give final feedback now but think the choices made are reasonable.

Reply:
They did not complain about the handling of the 0% case.